test/com/arpnetworking/metrics/portal/scheduling/impl/NeverScheduleTest.java

```diff
@@ -0,0 +1,21 @@
+package com.arpnetworking.metrics.portal.scheduling.impl;
+
+import org.junit.Test;
+
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+
+public class NeverScheduleTest {
```

Comment:
- Please include some bull-shit Javadoc
 - Class should be `final`.

e.g. https://github.com/ArpNetworking/metrics-aggregator-daemon/blob/master/src/test/java/com/arpnetworking/tsdcore/model/QuantityTest.java#L21